src/compiler/utilities.ts

```diff
@@ -4544,9 +4544,9 @@ export function isPushOrUnshiftIdentifier(node: Identifier) {
  *
  * @internal
  */
-export function isParameterDeclaration(node: Declaration): boolean {
+export function isPartOfParameterDeclaration(node: Declaration): boolean {

```

Comment:
I personally preferred `rootDeclarationIsParameter` or similar, but this was the suggestion so I just chose it.